moveit_ros/planning/planning_scene_monitor/demos/demo_scene.cpp

```diff
@@ -75,23 +85,28 @@ void sendKnife()
   co.primitive_poses[0].position.z = -0.2;
   co.primitive_poses[0].orientation.w = 1.0;
 
-  pub_aco.publish(aco);
+  pub_aco->publish(aco);
   sleep(1);
-  pub_aco.publish(aco);
-  ROS_INFO("Object published.");
-  ros::Duration(1.5).sleep();
+  pub_aco->publish(aco);
+
+  RCLCPP_INFO(node->get_logger(), "Object published.");
+  rclcpp::sleep_for(std::chrono::milliseconds(1500));
```

Comment:
There's also std::chrono::seconds

Reply:
We can't use double to initialize `std::chrono::seconds`, so `std::chrono::seconds(1.5)` will produce error

Reply:
Right, still weird that this seems to be the shortest way to implement a sleep right now.